nomad/structs/services.go

```diff
@@ -388,6 +388,11 @@ func (sc *ServiceCheck) validateNomad() error {
 		return fmt.Errorf("tls_server_name may only be set for Consul service checks")
 	}
 
+	// tls_skip_verify is consul only
+	if sc.TLSSkipVerify {
+		return fmt.Errorf("tls_skip_verify may only be set for Consul service checks")
```

Comment:
The error isn't using formatting, so we could probably just use `errors.New`.

Reply:
Ops, copy-pasta. I adjusted the others as well in https://github.com/hashicorp/nomad/pull/18333/commits/37b9a2d4614b69bc0b58d4ac930201db782ebe98